lib/goliath/request.rb

```diff
@@ -218,6 +218,24 @@ def post_process(results)
       end
     end
 
+    # Used to determine if the connection should  be kept open
+    #
+    # @return [Boolean] True to keep the connection open, false otherwise
+    def keep_alive
+      return false if @env[:terminate_connection]
+      case @env[HTTP_VERSION]
+        # HTTP 1.1: all requests are persistent requests, client
+        # must send a Connection:close header to indicate otherwise
+      when '2.0', '1.1' then
```

Comment:
2.0 is a noop here, we can remove it as it'll only create confusion. To wire up h2 support we need to plumb http-2 parser logic, etc.